src/chrome/content/rules/advenacs.com.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	advenacs.com is the primary operating domain of the Advena Network Protection Service.
+-->
+
+<ruleset name="Advena">
+        <target host="advenacs.com" />
+        <target host="secure.advenacs.com" />
+        <target host="verify.advenacs.com" />
+        <target host="*.advenacs.com" />
```

Comment:
Our styleguide recommends to avoid wildcards. Is there a special need for it?
Otherwise blog.advenacs.com seems to be an additional target.